xrpl/asyncio/transaction/reliable_submission.py

```diff
@@ -107,3 +115,103 @@ async def send_reliable_submission(
     return await _wait_for_final_transaction_outcome(
         transaction_hash, client, prelim_result, transaction.last_ledger_sequence
     )
+
+
+def _decode_tx_blob(tx_blob: str) -> Transaction:
```

Comment:
This seems like something that should be on the `Transaction` object - something like `Transaction.from_blob`